reference-architecture/azure-ansible/README.md

```diff
@@ -0,0 +1,75 @@
+# RedHat Openshift Enterprise cluster on Azure
+
+When creating the RedHat Openshift Enterprise Cluster on Azure, you will need a SSH RSA key for access.
+
+## SSH Key Generation
+
+1. Windows - https://www.digitalocean.com/community/tutorials/how-to-create-ssh-keys-with-putty-to-connect-to-a-vps
+2. Linux - https://help.ubuntu.com/community/SSH/OpenSSH/Keys#Generating_RSA_Keys
```

Comment:
No preference on a single link. I was trying to subtly suggest using our own documentation (from OpenShift Docs or Red Hat Docs) instead of a link from ubuntu.